src/remove_background.h

```diff
@@ -13,8 +14,11 @@ cv::Mat createLightPattern(cv::Mat image);
 
 cv::Mat binarizeImage(cv::Mat image);
```

Comment:
Actually after refactor we can remove these functions because `binarizeInamgeOtsu()` does the job for us